compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -233,10 +233,9 @@ pub enum TypeKind {
     Pointer = 12,
     Vector = 13,
     Metadata = 14,
-    X86_MMX = 15,
```

Comment:
Don't think we should remove it from the LLVM FFI API...

Reply:
I agree but on the other hand it's basically dead code.

Reply:
This enum is meant to match `LLVMTypeKind` in the LLVM C API.